projects/plugins/jetpack/tools/webpack.config.extensions.js

```diff
@@ -268,4 +301,53 @@ module.exports = [
 			} ),
 		],
 	},
+	{
+		...sharedWebpackConfig,
+		entry: {
+			...editorSingleBlocksScripts,
+			...viewSingleBlocksScripts,
+		},
+		plugins: [
+			...sharedWebpackConfig.plugins,
+			new CopyWebpackPlugin( {
+				patterns: [
+					{
+						from: '**/block.json',
+						to: '[path][name][ext]',
+						context: path.join( __dirname, '../extensions/blocks' ),
+					},
+				],
+			} ),
+		],
+	},
+	{
```

Comment:
Build styles by directly compiling SCSS files instead of importing styles in JS components. I thought this might fix the styles being loaded several times in the editor, but this might not be necessary eventually. I'll come back to this.